index.html

```diff
@@ -18,5 +18,69 @@
   <body>
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-  </body>
+  
+      <header class="header">
+
+        <a href="#">
+          <img
+            src="css/arsenal logo 2.png"
+            alt="Logo Arsenal"
+            class="header__logo"
+          />
+        </a>
+
+       <nav>
+          <ul class="navigation__list">
+            <li class="navigation__item">
+              <a class="navigation__link" href="#">Home Games</a>
+            </li>
+            <li class="navigation__item">
+              <a class="navigation__link" href="#">Away Games</a>
+            </li>
+            <li class="navigation__item">
+              <a class="navigation__link" href="#">Store</a>
+            </li>
+            <li class="navigation__item">
+              <a class="navigation__link" href="#">Contact Us</a>
+            </li>
+          </ul>
+        </nav>
+
+      
+      </header>
+      <main>
+        <section class="fans">
+          <h1 class="fans">Arsenal Home Fans Page</h1>
+          <h3 class="history">
+            Arsenal was the first club from the South of England to join the Football League in 1893, and they reached the First Division in 1904. Relegated only once, in 1913, they continue the longest streak in the top division, and have won the second-most top-flight matches in English football history..
+          </h3>
+          
+        </section>
+
+
+        <section class="info">
+              <h2 class="info-title">Best Football Club</h2>
+             
+                  
+                  
+          </section>
```

Comment:
You need to make use of the `article` tag - perhaps add a couple paragraphs explaining how the club was formed, recent successes, etc e.g.
```
<div>
   <article>
      <h2>Formation</h2>
      <p>the club was formed by x y z in 1200 BC</p>
   </article>
   <article>
      ......
   </article>
</div>
```